autogpt_platform/backend/backend/api/features/onboarding_dump/eval_corpus.py

```
@@ -0,0 +1,334 @@
+
+
+def _ffmpeg(*args: str) -> None:
+    subprocess.run(
+        ["ffmpeg", "-hide_banner", "-loglevel", "error", "-y", *args],
+        check=True,
+        capture_output=True,
+    )
+
+
+def _ffmpeg_stderr(*args: str) -> str:
+    result = subprocess.run(
```

Ruff (0.16.1)

[warning] `subprocess.run` without explicit `check` argument

Add explicit `check=False`

(PLW1510)